habitat-hitl/habitat_hitl/_internal/config_helper.py

```diff
@@ -98,7 +99,12 @@ def update_config(
                 if measurement_name in task_config.measurements:
                     task_config.measurements.pop(measurement_name)
 
-            sim_sensor_names = ["head_depth", "head_rgb"]
+            # todo: decide whether to fix up config here versus validate config
+            sim_sensor_names = [
```

Comment:
In the long run, I think it's hacky to "fix up" yaml configs at runtime. It creates confusion, because the config used at runtime doesn't match what the user authored in the yaml. It would be cleaner to instead just assert on the requirements here (thus forcing the user to fix up their config yaml).